src/utils/telemetry/index.js

```diff
@@ -45,36 +45,17 @@ const eventConfig = {
   ],
 }
 
-const track = async function (eventName, payload) {
-  const properties = payload || {}
-
-  if (IS_INSIDE_CI) {
-    if (DEBUG) {
-      console.log('Abort .identify call inside CI')
-    }
-    return Promise.resolve()
+const track = async function (eventName, payload = {}) {
+  if (isCI) {
+    return
   }
 
   const globalConfig = await getGlobalConfig()
-  // exit early if tracking disabled
-  const TELEMETRY_DISABLED = globalConfig.get('telemetryDisabled')
-  if (TELEMETRY_DISABLED && !properties.force) {
-    if (DEBUG) {
-      console.log('Abort .track call TELEMETRY_DISABLED')
-    }
-    return Promise.resolve()
+  if (isTelemetryDisabled(globalConfig)) {
+    return
   }
 
-  let userId = properties.userID
-  let { cliId } = properties
-
-  if (!userId) {
-    userId = globalConfig.get('userId')
-  }
-
-  if (!cliId) {
-    cliId = globalConfig.get('cliId')
-  }
+  const [userId, cliId] = [globalConfig.get('userId'), globalConfig.get('cliId')]
```

Comment:
I couldn't find a place we ass `track` `userId` and `cliId` as properties of `payload`, so we can always take them from `globalConfig